src/geouned/GEOUNED/core.py

```diff
@@ -681,8 +681,8 @@ def start(self):
                 if c.Definition.level == 0 or c.IsEnclosure:
                     continue
                 logger.info(f"simplify cell {c.__id__}")
-                Box = UF.get_box(c)
-                CT = build_c_table_from_solids(Box, (c.Surfaces, Surfs), option="full")
+                Box = UF.get_box(c,self.options)
```

Comment:
```suggestion
                Box = UF.get_box(c,self.options.enlargeBox)
```
and change the argument in ```get_box function``` to ```enlargeBox```